test/tools/libtesteth/ImportTest.cpp

```diff
@@ -181,12 +182,11 @@ bytes ImportTest::executeTest()
 			testObj["blocks"] = blocksArr;
 			json[testname] = testObj;
 
-			//testName() is changed during the execution of bctest!!!
-			fs::path const tmpFillerName = getTestPath() / fs::path("src/GenStateTestAsBcTemp") / fs::path(TestOutputHelper::caseName()) / fs::path(testname + "Filler.json");
-			writeFile(tmpFillerName, asBytes(json_spirit::write_string((json_spirit::mValue)json, true)));
-			dev::test::executeTests(testname, "/BlockchainTests/GeneralStateTests/" + TestOutputHelper::caseName(),
-											 "/GenStateTestAsBcTemp/" + TestOutputHelper::caseName(), dev::test::doBlockchainTestNoLog);
-
+			//Write a blockchain test .json file from dynamic filler 'json' that is produced from state test filler
+			BCGeneralStateTestsSuite genSuite;
+			fs::path const testFile = genSuite.getFullPath(TestOutputHelper::caseName()) / fs::path(testname + ".json");
+			json_spirit::mValue const generatedJson = genSuite.doTests((json_spirit::mValue)json, true);
+			writeFile(testFile, asBytes(json_spirit::write_string(generatedJson, true)));
```

Comment:
Why is `executeTests` not used anymore?  What happened to `GenStateTestAsBcTemp`?